Remotion/Data/DomainObjects.UnitTests/DataManagement/DataContainerTest.cs

```diff
@@ -186,7 +187,7 @@ public void GetValue_InvalidProperty ()
     {
       Assert.That (
           () => _existingDataContainer.GetValue (_nonOrderProperty), 
-          Throws.ArgumentException.With.Message.Contains ("Parameter name: propertyDefinition"));
+          Throws.ArgumentException.With.Message.Contains ("propertyDefinition"));
```

Comment:
See ArgNullMessage-check  / helper method

Reply:
What if the message was also validated? This would prevent the parameter from being checked without context.

It would also be possible to replace the `Contains` with  `Matches`, the regex could look something like this `(Parameter) .+(propertyDefinition)`. This would work in both versions and would also prevent the parameter being tested without context

Reply:
The regex would mean hardcoding the net45 / net5.0 syntaxes, which we are trying to avoid.

validating the message: possibly, question is, why was this avoided before?

Third option: what does ArgumentException say to passing an empty string as the message? Do we then get only a paramtername-string in the resulting message we could assert with contains?

Reply:
> Third option: what does ArgumentException say to passing an empty string as the message? Do we then get only a paramtername-string in the resulting message we could assert with contains?

this `new ArgumentException (message: string.Empty, paramName: "propertyDefinition").Message` returns `\r\nParameter name: propertyDefinition`, i think this the best solution